Sources/StreamVideoSwiftUI/CallView/VisibilityThresholdModifier.swift

```diff
@@ -100,12 +100,13 @@ extension View {
         threshold: CGFloat = 0.3,
         changeHandler: @escaping (Bool) -> Void
     ) -> some View {
-        modifier(
-            VisibilityThresholdModifier(
-                in: bounds,
-                threshold: threshold,
-                changeHandler: changeHandler
-            )
-        )
+//        modifier(
```

Comment:
would prefer if we keep this one in, instead of using onAppear/Disappear.

Reply:
We will for sure keep it. I have spent too much time tweaking it. Leaving this comment unresolved so we have a reference and be able to restore in a subsequent PR